test/test-pages/nytimes-1/expected.html

```diff
@@ -1,36 +1,47 @@
+                <p data-para-count="231" data-total-count="3365">Officials divulged Thursday that the Sudanese government had allowed two visits by American operatives to a restricted border area near Libya, which they cited as evidence of a new spirit of cooperation on counterterrorism efforts.</p>
+                <p data-para-count="248" data-total-count="3613" id="story-continues-2">In addition to continuing violence in Darfur, several other serious conflicts are raging in southern and central Sudan, along with a civil war in newly independent South Sudan, which Sudan has been suspected of inflaming with covert arms shipments.</p>
+                <p data-para-count="145" data-total-count="3758"><a href="http://sudanreeves.org/2016/07/04/eric-reeves-is-now-a-senior-fellow-at-harvard-universitys-francois-xavier-bagnoud-center-for-health-and-human-rights/">Eric Reeves</a>, one of the leading American academic voices on Sudan, said he was “appalled” that the American government was lifting sanctions.</p>
+                <p data-para-count="215" data-total-count="3973">He said that Sudan’s military-dominated government continued to commit grave human rights abuses and atrocities, and he noted that just last week Sudanese security services <a href="http://sudanreeves.org/2017/01/02/7710/">killed</a><a href="http://sudanreeves.org/2017/01/02/7710/"> more than 10 civilians in Darfur</a>.</p>
+                <p data-para-count="250" data-total-count="4223">“There is no reason to believe the guys in charge have changed their stripes,” said Mr. Reeves, a senior fellow at the François-Xavier Bagnoud Center for Health and Human Rights at Harvard University. “These guys are the worst of the worst.”</p>
+                <p data-para-count="208" data-total-count="4431">Obama administration officials said that they had briefed President-elect Donald J. Trump’s transition team, but that they did not know if Mr. Trump would stick with a policy of warmer relations with Sudan.</p>
+                <p data-para-count="143" data-total-count="4574">They said that Sudan had a long way to go in terms of respecting human rights, but that better relations could help increase American leverage.</p>
+                <p data-para-count="149" data-total-count="4723" data-node-uid="1">Mr. Reeves said he thought that the American government was being manipulated and that the Obama administration had made a “deal with the devil.”</p>
+                <p><a href="#whats-next">Continue reading the main story</a>
+                </p>
+            </div>
+        </article>
+    </main>
+    <section id="site-index">
+        <nav id="site-index-navigation" role="navigation">
+        </nav>
+    </section>
```

Comment:
Hm, is this being pulled in because of the ancestor scoring?

Reply:
I think so, yes. Maybe [this](https://github.com/mozilla/readability/blob/master/Readability.js#L895) could be extended to cover role=navigation (removing the nav element), and then the section below that would remove the empty section element?

Reply:
Yeah, maybe - let's use a new issue / pr for that though...

Reply:
Filed #616.